awslimitchecker/checker.py

```diff
@@ -83,6 +83,12 @@ def __init__(self, warning_threshold=80, critical_threshold=99,
           com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html>`_
           string to use when assuming a role via STS.
         :type external_id: str
+        :param mfa_serial_number: (optional) the `MFA Serial Number` string to_
+        use when assuming a role via STS.
```

Comment:
The continuation line here and on line 89 needs to be indented two spaces. That's why the [Travis doc test](https://travis-ci.org/jantman/awslimitchecker/jobs/95972816#L278) is failing.